src/components/Input.js

```diff
@@ -241,7 +247,7 @@ class Input extends Component {
             colour={colour}
             onFocus={this._onFocus}
             onBlur={this._onBlur}
-            onChange={this._onChange}
+            onChange={e => this._onChange(e, onChange)}
```

Comment:
you don't need to pass onChange here I think, can just be accessed as `this.props.onChange` inside `_onChange` directly I think?